vispy/geometry/__init__.py

```diff
@@ -19,5 +19,6 @@
 from .torusknot import TorusKnot  # noqa
 from .calculations import (_calculate_normals, _fast_cross_3d,  # noqa
                            resize)  # noqa
-from .generation import create_arrow, create_cone, create_cube, \
-           create_cylinder, create_sphere  # noqa
+from .generation import (create_arrow, create_box, create_cone,  # noqa
+                         create_cube, create_cylinder, create_plane,  # noqa
+                         create_sphere)  # noqa
```

Comment:
when adding things to `__init__` you need to also update `__all__` above, as it's used to set the order when creating docs and to clean up imports